testsuite/forge.py

```diff
@@ -1371,6 +1371,7 @@ def test(
     test_args: Optional[List[str]],
     test_suites: Tuple[str],
 ) -> None:
+    forge_test_suite = "validators_join_and_leave"
```

Comment:
remember to remove this

Reply:
Ahhhh this is causing forge-continuous to run which is trying to run with an old forge / validator image which then fails

So when you remove this it will work